imagelib/admin/thumbnailbuilder.php

```diff
@@ -130,20 +132,20 @@ function resetRebuildForm(f){
 						<form name="tnbuilderform" action="thumbnailbuilder.php" method="post">
 							<div class="fieldRowDiv">
 								<div class="fieldDiv">
-									<input name="buildmed" type="checkbox" value="1" <?php echo ($buildMediumDerivatives?'checked':''); ?> />
-									<span class="fieldLabel"> <?php echo $LANG['INCLUDE_MED']; ?></span>
+									<input id="buildmed" name="buildmed" type="checkbox" value="1" <?php echo ($buildMediumDerivatives?'checked':''); ?> />
+									<label for="buildmed" class="fieldLabel"> <?php echo $LANG['INCLUDE_MED']; ?> </label>
 								</div>
 							</div>
 							<div class="fieldRowDiv">
 								<div class="fieldDiv">
 									<input name="evalorientation" type="checkbox" value="1" <?php echo ($evaluateOrientation?'checked':''); ?> />
```

Comment:
I think this needs an `id="evalorientation"` attribute for the label to be recognized? I just tested this on Total Validator Pro, and it seems to be true.